tests/test_sql_utils.py

```diff
@@ -97,59 +97,3 @@ def test_first_dim_select():
     res = first_dim_select(["os", "locale"], indent=0).strip()
     shouldbe = ", first(os) as os\n, first(locale) as locale"
     assert res == shouldbe
-
-
-def test_base_query():
```

Comment:
Is this not needed because of the newer `test_bgbb/test_fit_airflow_job_cli` test?

Reply:
This was removed because it's too strict to be useful. Once I started to modify the SQL query itself, the test failures here get in the way. The other tests that implicitly use `mk_rec_freq_q` should be enough to verify that this query is correct.